rskj-core/src/main/java/co/rsk/db/StateRootTranslator.java

```diff
@@ -20,20 +20,21 @@
 import co.rsk.crypto.Keccak256;
 import org.ethereum.datasource.KeyValueDataSource;
 
+import java.util.HashMap;
 import java.util.Map;
 
 public class StateRootTranslator {
```

Comment:
This `StateRootTranslator` is looking a lot like a DataBaseWithCache after the simpleification.

Reply:
good point, re-used `DataSourceWithCache`